lib/elixir/lib/task.ex

```diff
@@ -187,7 +187,18 @@ defmodule Task do
   Awaits a task reply.
 
   A timeout, in milliseconds, can be given with default value
-  of `5000`. In case the task process dies, this function will
+  of `5000`. In the event of a timeout handle it with try 
+  catch statement. 
+  
+      timeout = 10
+      try do
+        Task.await(search_task, timeout)
+      catch
```

Comment:
I would prefer if we don't catch the raise. Why is your rationale here? Do you want to check if the task was completed? @fishcakez anything we can do to just check?